core/notifications/src/history/repo.rs

```diff
@@ -74,19 +74,20 @@ impl PersistentNotifications {
         Ok(())
     }
 
-    pub async fn unacknowledged_notifications_count(
+    pub async fn count_unacknowledged_notifications(
         &self,
         user_id: GaloyUserId,
-    ) -> Result<i64, NotificationHistoryError> {
+    ) -> Result<u64, NotificationHistoryError> {
         let count = sqlx::query_scalar!(
-            r#"SELECT COUNT(*)
+            r#"SELECT COUNT(*) as "count!"
             FROM stateful_notifications
             WHERE galoy_user_id = $1 AND acknowledged = FALSE"#,
             user_id.as_ref(),
         )
         .fetch_one(self.read_pool.inner())
         .await?
-        .expect("count can not be None");
+        .try_into()
```

Comment:
an just use `as u64`

Reply:
yeah thought about that as well but I think I want to go for expect... do you have a strong reason not to use it ?